File: green-third-party-sdk/green-csharp-sdk/green-csharp-sdk/ImageSyncScanSample.cs
using System;
using Model;
using AliYunCore;
using Utils;
using Service;

namespace greenImageCsharpsdk
{
    class MainClass
    {
        public static void Test()
        {
            // create your profile 
            DefaultProfile profile = new DefaultProfile("cn-shanghai", "<your access key id>", "<your access key secret>");
            AliYunClient client = new AliYunClient(profile);

            ClientInfo clientInfo = new ClientInfo();
            clientInfo.ip = "127.0.0.1";

            // image scan task
            Task imageTask = new ImageTask("https://xxx.png");
            imageTask.dataId = Guid.NewGuid().ToString();
            BizData imageBizData = new BizData("Image", new String[] { "porn" }, new Task[] { imageTask });

            string algoPath = "/green/image/scan";
            Request req = new Request(imageBizData, algoPath);
            req.addQueryParameter("clientInfo", JSON.stringify(clientInfo));

            string response = client.getResponse(req);

            // your biz code
            Console.WriteLine(response);

            Console.ReadKey();
        }
    }
}

File: green-third-party-sdk/green-cpp-sdk/ImageSyncScanSample.cc
#include <iostream>
#include "GreenService.h"

using namespace std;
using namespace AliGreen;
using namespace AliYunCore;

int main()
{
    DefaultProfile profile( "cn-shanghai", "<your access key id>",
        "<your access key secret>" );

    GreenClient client(profile);

    string algoPath = "/green/image/scan";

    ClientInfo clientInfo = ClientInfo();
    clientInfo.addClientInfoItem("ip", "127.0.0.1");
    clientInfo.addClientInfoItem("os", "Android");


    list<string> scenes;
    scenes.push_back("porn");
    scenes.push_back("ad");


    list<Task*> tasks;
    Task* task1 = new ImageTask("https://xxx.png");
    task1->setDataId(MakeRand());

    tasks.push_back(task1);

    BizData bizData = BizData("Test", scenes, tasks);

    Request* req = new ImageRequest(bizData, algoPath, clientInfo);

    string response = client.getResponse(req);

    // your biz code
    cout << "res:" << response << endl;

    return 0;
}
